plugins/grpc/forwarder.go

```diff
@@ -345,20 +345,32 @@ func (g *GRPCForwarder) SchedulerEvent(ctx context.Context, infos, fwdMetadata m
 
 //Forward send room or player status to specified server
 func (g *GRPCForwarder) Forward(ctx context.Context, event string, infos, fwdMetadata map[string]interface{}) (status int32, message string, err error) {
+
+	// Add forwarder metadata (from maestro config) to request
+	// Client metadata should take priority
+	metadata := g.config.GetStringMap("metadata")
+	if metadata != nil {
+		for k := range fwdMetadata {
+			metadata[k] = fwdMetadata[k]
+		}
+	} else {
+		metadata = fwdMetadata
+	}
+
 	l := g.logger.WithFields(log.Fields{
 		"op":          "Forward",
 		"source":      "plugin/grpc",
 		"event":       event,
 		"infos":       fmt.Sprintf("%v", infos),
-		"fwdMetadata": fmt.Sprintf("%v", fwdMetadata),
+		"fwdMetadata": fmt.Sprintf("%v", metadata),
```

Comment:
logging this for every event isn't excessive? why is this info instead of debug?

Reply:
up